doc/api/cluster.markdown

```diff
@@ -302,30 +643,30 @@ because of exiting or being signaled). Otherwise, it returns `false`.
 * `signal` {String} Name of the kill signal to send to the worker
   process.
 
-This function will kill the worker. In the master, it does this by disconnecting
-the `worker.process`, and once disconnected, killing with `signal`. In the
-worker, it does it by disconnecting the channel, and then exiting with code `0`.
+This function will, by default, signal the worker process to terminate. In the
+master process, this is done by first disconnecting the worker process, then
+sending the `signal` (which defaults to `SIGTERM`). Within the worker, the IPC
```

Comment:
This is a common source of bug reports: since after disconnect all servers have been closed, the worker usually exits, so the signal is not delivered. I think it should be more clear here that while another signal can be used, it should be expected that the signal is NOT usually delivered, and is only a backup for if the disconnect failed to cause normal exit. If you really want to use signals for IPC, not terminating the worker, then use worker.process.kill()